webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
Also, of the three changes in this patch, I think @ShijunS has a point on this one that it's perhaps unwise to dictate substantive steps involving non-stack objects after requiring the UA to synchronously execute content script code, as it's a window to potentially trip up assumptions in the remaining steps in some implementations. For instance, an implementation that didn't properly root the peer connection object on the stack in this queued task, might not take kindly to content script removing its last peer connection reference. I believe that's what he's referring to as reentrancy.